### ISSUE TITLE
Can't specify custom configuration for linker

### ISSUE THREAD

**Issue by moledzki** [rolandshacks/vs64#50]
Hi,

I can't use custom configuration file for linker because --config and -t options can't be enaled together.
It seems that vs64 always add -t flag even when --config (or -C) option is defined.
So after defining linkerFlags in config file:

 ```
"linkerFlags": [
        "--config",
        "myconfig.cfg"
    ],
```
ld65 is invoked with both --config and -t flag.

```
ld65 (...) --config myconfig.cfg -t c64
ld65: Error: Cannot use -C/-t twice
```

**Comment by rolandshacks** [comment]
Thanks for the feedback. Seems an easy thing to add, let me see if I can find some time for it soon.

**Comment by rolandshacks** [comment]
The issue should be fixed with the version 2.4.8

Please download and manually install from here: https://github.com/rolandshacks/vs64/releases/download/v2.4.8/vs64-2.4.8.vsix

**Comment by moledzki** [comment]
Thanks, it's working now.
